DataTable/ColumnBuilder.php

```diff
@@ -12,25 +11,20 @@
  */
 class ColumnBuilder
 {
-    protected $columns;
+    protected array $columns;
 
     /**
      * @param string $colidentifier
      * @param ColumnInterface $definition
-     *
-     * @return ColumnBuilder
      */
-    public function add($colidentifier, ColumnInterface $definition)
+    public function add(string $colidentifier, ColumnInterface $definition)
```

Comment:
please always be careful with this kind of change, check if it can be never null otherwise it is a fatal error